test/runtests.jl

```diff
@@ -10,12 +10,13 @@ const is_TRAVIS = haskey(ENV,"TRAVIS")
 if GROUP == "All" || GROUP == "Interface"
     @time @safetestset "Utilities Tests" begin include("utils.jl") end
     @time @safetestset "Poisson example" begin include("../examples/poisson.jl") end
-    @time @safetestset "Heat equation example" begin include("../examples/heat_equation.jl") end
+#     @time @safetestset "Heat equation example" begin include("../examples/heat_equation.jl") end
```

Comment:
why is this commented out? Tests should almost never be commented out and instead set as `@test_broken` as needed.

Reply:
I forgot to uncomment this test; apologies!